modules/dxp/apps/segments/segments-asah-connector-test/src/testIntegration/java/com/liferay/segments/asah/connector/internal/constants/AnalyticsConnectionPropertiesConstants.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.segments.asah.connector.internal.constants;
+
+/**
+ * @author Mikel Lorza
+ */
+public class AnalyticsConnectionPropertiesConstants {
```

Comment:
There are not any file named *PropertiesConstants in the whole portal, what do you think if you create an AnalyticsConnectionImplementation, of a stratic Map in some TestUtil class?